P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,252 @@
+                the standard level of indentation which should be encoded as indentLevel="0").
+                Basically used for Wiki talk pages, but may and should also be used for genres such
+                as webblog comments when the source was HTML.
+                <!--[Adapted from the <ref target="#TEIcmc2012"
+                    >DeRiK-Schema</ref>].-->
+            </desc>
+            <datatype minOccurs="1" maxOccurs="1">
+                <dataRef key="teidata.count"/>
+            </datatype>
+        </attDef>
+    </attList>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>standard user postings in chat</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post type="standard" generatedBy="human" who="#A05" rend="color:red">
+                <p>oh ja, in die Wärme...</p>
+            </post>
+            <post type="standard" generatedBy="human" who="#A03" rend="color:red">
+                <p>nein, hast du nicht !!</p>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>chat message-type "system": user enters a chatroom, "system" defined in listPerson</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <listPerson>
+                <person xml:id="system">
+                    <persName>system</persName>
+                </person>
+                <!-- [...] -->
+            </listPerson>
+            <!--  [...] -->
+            <post type="event" generatedBy="system" who="#system" rend="color:navy">
+                <p>
+                    <name type="nickname" corresp="#A07">Interseb</name> betritt den Raum.</p>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>action message in chat</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post type="event" generatedBy="human" who="#A02" rend="color:blue">
+                <p>
+                    <name type="nickname" corresp="#A02">McMike</name> hechtet zum Steuer </p>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Chat2CLARIN" xml:lang="de" versionDate="2021-09-07">
+        <p>chat messages incl. interaction words (types: inflective and acronym) and an interaction
+            phrase</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post type="standard" generatedBy="human" who="#A03" rend="color:green">
+                <p>
+                    <w type="AKWIND">*</w><w type="AKW" subtype="inflective">schwank</w><w
+                        type="AKWIND">*</w>
+                    <!--...-->
+                    <w type="AKWIND">*</w><w type="AKW">seekrank</w><w type="AKWIND">*</w>
+                </p>
+            </post>
+            <!--[...] -->
```

Comment:
please do not use the brackets here, just: `<!-- ... -->`

As a rule: 
When omitting elements use: `<!-- ... -->`
When omitting text inside an element use: [...]

(The combination of square brackets with an XML comment should nowhere appear. Yes, there is an old ticket for that... https://github.com/TEIC/TEI/issues/1961)

Reply:
@sabineseifert  — We have 468 occurrences of “...” inside text() nodes inside `<egXML>`, of which only 64 are surrounded by square brackets. So I am going with just “...” without square brackets for now. If you decide that they should all have square brackets there will be ½ dozen more to fix when you attack #1961, sorry.

Reply:
fixed at f0e7d1e26.